### PR TITLE
Avoid unnecessary else statement

### PR DESCRIPTION
**Pull request by WouterSioen**
It's a bit more readable imo.

### CODE REVIEW

File: src/StatFetcher.php [hunk 1]
```diff
@@ -34,10 +34,10 @@ public function __construct(
         $this->package = $package;
 
         if (is_null($repository)) {
-            $this->repository = new NpmRegistryRepository(ClientBuilder::Build());
-        } else {
-            $this->repository = $repository;
+            $repository = new NpmRegistryRepository(ClientBuilder::Build());
         }
+        
+        $this->repository = $repository;
```
Comment (lcobucci) [inline src/StatFetcher.php:40]: Why not simply this:

```php
$this->repository = $repository ?: new NpmRegistryRepository(ClientBuilder::Build());
```

Then you can remove the `if`.

**Comment by vanbrabantf** [comment]
Thanks @WouterSioen it's indeed better. But I have to agree with @lcobucci, the inline one if even better :).

I'm going to make a PR that implements the inline if.

**Comment by WouterSioen** [comment]
👌🏻

**Comment by vanbrabantf** [comment]
Still thanks tho 😗

**Pull request closed without merging**